src/shared/components/horizontal-scroller/arrows/index.module.css

```diff
@@ -0,0 +1,76 @@
+@import "shared/styles/imports/variables.css";
+@import "shared/styles/imports/custom-medias.css";
+
+:root {
+  --color-white-low-opacity: rgba(255, 255, 255, 0.2);
+  --color-white-medium-opacity: rgba(255, 255, 255, 0.4);
+  --button-size: 45px;
+}
+
+.container {
+  position: absolute;
+  top: calc(50% - (var(--button-size) / 2));
```

Comment:
could you add a small comment explaining these calcs? Would help on reading this